ext/RMagick/rmimage.c

```diff
@@ -12076,6 +12076,14 @@ resample(int bang, int argc, VALUE *argv, VALUE self)
 
     exception = AcquireExceptionInfo();
 #if defined(IMAGEMAGICK_7)
+    if (blur > 1.0)
+    {
+        image = BlurImage(image, blur, blur, exception);
+    }
+    else if (blur < 1.0)
+    {
+        image = SharpenImage(image, blur, blur, exception);
```

Comment:
@Watson1978 can you explain more about your solution here?

Reply:
ImageMagick 6 has a behavior to blur/sharp.

```
%    o blur: the blur factor where > 1 is blurry, < 1 is sharp.
```

https://github.com/ImageMagick/ImageMagick6/blob/2d96228eec9fbea62ddb6c1450fa8d43e2c6b68a/magick/resize.c#L2275